code/game/machinery/computer/arcade/arcade.dm

```diff
@@ -35,7 +35,6 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		/obj/item/toy/talking/owl = 2,
 		/obj/item/toy/talking/griffin = 2,
 		/obj/item/coin/antagtoken = 2,
-		/obj/item/stack/tile/fakespace/loaded = 2,
```

Comment:
Rather then removing this, after the maps have finished loading in `/datum/controller/subsystem/mapping`, check to see if any station z levels contain your new trait, and if they do, remove this type from `GLOB.arcade_prize_pool`